src/main/app/src/Components/Landscape/Map/MapRelation/MapRelation.tsx

```diff
@@ -10,49 +13,51 @@ interface Props {
 
 /**
  * Returns a choosen Map Relation
- * @param element Choosen SVG Element from our Landscape Component
+ * TODO work with items and relation objects
```

Comment:
What has to be done with this `TODO`?

And I noticed that it is really difficult to click on the relation. I am quite sure, that an user would not have a clue that the relations are clickable. I just came across the fact that it is clickable, because I looked up how `MapRelation` is used in the code.

1. The lines must be thicker in order to be clickable properly
2. A hint or something must exist to invite the user to click them, e.g. also change the mouse to the hand symbol when hovering over the (thicker) lines

Reply:
The TODO says that there are objects for items and relations available (loaded via API), but these are not used in here.

Regarding map relations: https://github.com/dedica-team/nivio/issues/348

Reply:
The hand symbol is working, nice!